apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/AnyAssertionSamples.kt

```diff
@@ -201,25 +201,22 @@ class AnyAssertionSamples {
         }
     }
 
+    interface Person {
+        val age: Int
+    }
+
+    private val customers = emptyList<Person>()
+
     @Test
     fun because() {
-        expect(12).because("this is true as anything can ever be") {
-            toBe(12)
-        }
-
-        fails {
-            expect(12).because("won't hold, it's not equal") {
-                toBe(11)
+        expect("filename?")
```

Comment:
I suggest you wrap everything into `fails`

Reply:
I made the examples succeed instead, is that alright?

Reply:
that's fine as well